devel.yaml.jinja

```diff
@@ -10,7 +10,7 @@ services:
       default:
       public:
     ports:
-      - "127.0.0.1:{{ macros.version_major(odoo_version) }}899:6899"
+      - "127.0.0.1:{{ macros.version_major(odoo_version) }}678:5678"
```

Comment:
Is there a real need for this? Because otherwise I prefer to just use port 6899 like until today. It's used by `pudb` in remote mode also, if you want to use that debugger.

Reply:
Ok. I removed the last commit with the changes